static/less/style.less

```diff
@@ -349,3 +354,45 @@ img {
 .float-left {
   float: left;
 }
+
+.inverted-colors {
+  color: @inverted-fg-color;
+  background-color: @inverted-bg-color;
+  a {
+    color: @inverted-fg-color;
+  }
+  p {
+    a {
+      color: @inverted-fg-color;
+      &:hover {
+        color: @inverted-link-color;
+      }
+    }
+  }
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6 {
+    color: @inverted-fg-color;
+  }
+  code {
+    background-color: @inverted-darker-fg-color;
+    color: @fg-color;
+  }
+
+  table td, table th {
+    border: 2px solid @inverted-darker-fg-color;
```

Comment:
Do the same as you did for `pre` here. You just need to define `border-color`.

Reply:
✅ thx